### ISSUE TITLE
Update Profile: Luiza Novytska

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/UNLV-FIA-Group#335]
This existing employee profile needs to be updated with the information below.

Employee Name: Luiza Novytska

### Information to Update:
- Job Description: Replace "Student Accessibility Tester" with "**Junior Database Administrator**"
- Biography: Replace: Replace the current bio with the one below:
> Luiza Novytska joined the UNLV-FIA group as a student Web Accessibility Tester in June 2024. Certified by the Department of Homeland Security as a Trusted Tester, her main responsibility was to assesses web applications for compliance with Web Content Accessibility Guidelines (WCAG). In January 2026, Luiza transitioned to a full-time role as a Junior Database Administrator. In this capacity, she will handle a range of database administration tasks, oversee release management, manage projects related to several FIA and national change management initiatives, and work on Section 508 Compliance projects.
Luiza’s academic background boasts two M.S. degrees, one in Hospitality and the other in Economics with a focus on International Affairs. She is currently pursuing her third M.S. in Data Analytics, with an anticipated graduation in the Spring of 2026. Beyond her professional pursuits, Luiza enjoys spending quality time with family, indulging in sewing projects, and exploring new destinations through travel. She is also a dedicated Star Wars fan..

### Important Notes:
- Ensure the new employee is added alphabetically to the list (leave Brenda & Gretchen as the first two no matter what)
- Ensure newly added images are similar in size (200x200) or that they are cropped/expanded evenly to fit size requirements. All images should be compressed and stored within the appropriate assets folder
- Ensure newly added divs, buttons, dialogs, etc. are given unique IDs (if an ID was necessary for that element)
- Ensure your updated employee profile hasn't disrupted the functionality/information of any existing employee
